### DISCUSSION TITLE
Compatability with python 3.10

### DISCUSSION THREAD

**Discussion by murtazahathiyari** [talmolab/sleap#1880] [Help!]
Hi,

I am trying to get SLEAP to work with other packages, one of which is constrained to work with python 3.10. Is there a way to install the package such that it can be compatible with that version of python in a virtual environment? Our main purpose is to run inference on the SLEAP models in this environment. 

Best,
Murtaza

**Comment by eberrigan** [comment]
Hi @murtazahathiyari,

What operating system are you on?

I am currently working on this for Windows (almost there!). It should work for Apple Silicon already. 

Thanks!

Elizabeth

**Comment by murtazahathiyari** [comment]
Hi Elizabeth,

I am currently working off Ubuntu 22.04. Are there plans to get it working for this OS?

Best,
Murtaza
